chromium_src/base/threading/thread_restrictions.h

```diff
@@ -6,20 +6,39 @@
 #ifndef BRAVE_CHROMIUM_SRC_BASE_THREADING_THREAD_RESTRICTIONS_H_
 #define BRAVE_CHROMIUM_SRC_BASE_THREADING_THREAD_RESTRICTIONS_H_
 
+#include "brave/browser/widevine/buildflags.h"
+
 class BraveBrowsingDataRemoverDelegate;
 namespace ipfs {
 class IpfsService;
 }
 namespace brave {
 class ProcessLauncher;
 }
+namespace component_updater {
+class WidevineCdmComponentInstallerPolicy;
+}
 
-#define BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H  \
-  friend class ::BraveBrowsingDataRemoverDelegate; \
-  friend class ipfs::IpfsService;                  \
+#define BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H_BASE \
+  friend class ::BraveBrowsingDataRemoverDelegate;     \
+  friend class ipfs::IpfsService;                      \
   friend class brave::ProcessLauncher;
 
+#if BUILDFLAG(WIDEVINE_ARM64_DLL_FIX)
+// WidevineCdmComponentInstallerPolicy needs to use TimedWait:
+#define BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H_EXTRA \
```

Comment:
let's name this to include widevine/arm64. something like:
`BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_WIDEVINE_ARM64_DLL_FIX`

because another optional item may appear here. I don't think we want to call them `EXTRA_2`, `EXTRA_3`...